ntc_templates/templates/mikrotik_routeros_routing_ospf_interface_print_terse.textfsm

```diff
@@ -0,0 +1,10 @@
+Value INTERFACE (\S+)
+Value DYNAMIC (D)
+Value PASSIVE (P|passive)
+Value COST (\d+)
+
+Start
+  ^\s*\d+\s*(${DYNAMIC}\s*)?(${PASSIVE}\s*)?(comment=.+?)?interface=${INTERFACE}\s*cost=${COST}\s*priority=\d+\s*authentication=\S+\s*authentication-key=(?:")?\S+(?:")?\s*authentication-key-id=\d+\s*network-type=\S+\s*instance-id=\d+\s*retransmit-interval=\d+s\s*transmit-delay=\d+s\s*hello-interval=\d+s\s*dead-interval=\d+s\s*use-bfd=\S+$$ -> Record
+  ^\s*\d+\s*(D\s*)?address=\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}%${INTERFACE}\s*area=\S+\s*state=(${PASSIVE}|\S+)\s*network-type=\S+\s*cost=${COST}\s*(priority=\d+\s*)?use-bfd=\S+\s*retransmit-interval=\d+s\s*transmit-delay=\d+s\s*hello-interval=\d+s\s*dead-interval=\d+s\s*$$ -> Record
```

Comment:
Do you want to capture the D for dynamic in the terse01 raw output?
If this helps, please click the button to accept my commit suggestion. Thank you.

```suggestion
  ^\s*\d+\s*(${DYNAMIC}\s*)?address=\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}%${INTERFACE}\s*area=\S+\s*state=(${PASSIVE}|\S+)\s*network-type=\S+\s*cost=${COST}\s*(priority=\d+\s*)?use-bfd=\S+\s*retransmit-interval=\d+s\s*transmit-delay=\d+s\s*hello-interval=\d+s\s*dead-interval=\d+s\s*$$ -> Record
```

Reply:
In RouterOS 7, all interfaces in the 'routing ospf interface print terse' output are marked with 'D' even though the interfaces are not actually dynamically added into OSPF (unlike RouterOS 6), hence why it's ignored.

Reply:
Thank you @kwood92 for the details.
I don't work with RouterOS much these days so I wasn't aware. :face_with_peeking_eye: